### ISSUE TITLE
[BUG] training `polar` model gives `nan` RMSE in lcurve.out

### ISSUE THREAD

**Issue by Yi-FanLi** [deepmodeling/deepmd-kit#4554]
### Bug summary

I am training a polarizability model by running the task provided in https://github.com/deepmodeling/deepmd-kit/blob/r3.0/examples/water_tensor/polar. However, the `lurve` is printing `nan` values for the loss function:

```
#  step      rmse_val    rmse_trn   rmse_lc_val rmse_lc_trn   rmse_gl_val rmse_gl_trn         lr                                    
# If there is no available reference data, rmse_*_{val,trn} will print nan                                                          
      0      3.99e-01    0.00e+00           nan         nan           nan         nan    1.0e-02                                    
    100      7.35e-01    0.00e+00           nan         nan      9.18e-02         nan    6.0e-03                                    
    200      4.88e-01    0.00e+00           nan         nan      6.10e-02         nan    3.6e-03                                    
    300      2.03e-01    1.86e-01           nan         nan           nan    2.32e-02    2.1e-03
    400      2.48e-01    3.35e-01           nan         nan           nan    4.19e-02    1.3e-03
    500      1.47e-01    2.20e-01           nan         nan           nan    2.75e-02    7.7e-04
    600      1.34e-01    2.66e-01           nan         nan           nan    3.32e-02    4.6e-04
    700      9.79e-02    1.98e-01           nan         nan           nan    2.47e-02    2.8e-04
    800      3.33e-01    2.43e-01           nan         nan           nan    3.04e-02    1.7e-04
    900      2.50e-01    2.18e-01           nan         nan           nan    2.73e-02    9.9e-05
   1000      1.29e-01    3.14e-01           nan         nan           nan    3.92e-02    5.9e-05
   1100      0.00e+00    0.00e+00           nan         nan           nan         nan    3.5e-05
   1200      2.82e-01    2.24e-01           nan         nan           nan    2.80e-02    2.1e-05
   1300      4.46e-01    0.00e+00           nan         nan      5.58e-02         nan    1.3e-05
   1400      1.34e-01    0.00e+00           nan         nan           nan         nan    7.6e-06
   1500      3.17e-01    2.20e-01           nan         nan           nan    2.75e-02    4.6e-06
   1600      2.88e-01    2.63e-01           nan         nan           nan    3.28e-02    2.7e-06
   1700      4.65e-01    3.01e-01           nan         nan      5.81e-02    3.77e-02    1.6e-06
   1800      1.48e-01    1.12e-01           nan         nan           nan    1.40e-02    9.8e-07
   1900      2.80e-01    0.00e+00           nan         nan           nan         nan    5.9e-07
   2000      2.92e-01    2.23e-01           nan         nan           nan    2.78e-02    3.5e-07
```

Is it a bug or a normal behavior?

### DeePMD-kit Version

DeePMD-kit v3.0.0

### Backend and its version

TensorFlow v2.17.0

### How did you download the software?

conda

### Input Files, Running Commands, Error Log, etc.

The task is taken from `https://github.com/deepmodeling/deepmd-kit/blob/r3.0/examples/water_tensor/polar`.
I used this command to run it:
`dp train polar_input.json`

### Steps to Reproduce

Expalained above.

**Comment by njzjz** [comment]
Duplicate of #4536
